test/common/stats/udp_statsd_test.cc

```diff
@@ -29,21 +35,73 @@ TEST_P(UdpStatsdSinkTest, InitWithIpAddress) {
   Network::Address::InstanceConstSharedPtr server_address =
       Network::Utility::parseInternetAddressAndPort(
           fmt::format("{}:8125", Network::Test::getLoopbackAddressUrlString(GetParam())));
-  UdpStatsdSink sink(tls_, server_address);
+  UdpStatsdSink sink(tls_, server_address, false);
   int fd = sink.getFdForTests();
   EXPECT_NE(fd, -1);
 
   // Check that fd has not changed.
+  std::vector<Tag> tags;
```

Comment:
I would recommend moving this into the base mock class, and then putting the ON_CALL `WillByDefault` into the constructor, so that it can be use by others later.

Reply:
Cool, how about this one? 85dab7fd3cc3fd37fee05752b6058e1851dc73d3